src/util/config.rs

```diff
@@ -740,15 +742,16 @@ mod check_data_dir {
             libc::endmntent(afile);
             if fs.mnt_dir.is_empty() {
                 return Err(ConfigError::FileSystem(format!(
-                    "path: {:?} not find in mountable",
-                    path
+                    "{}: path: {:?} not find in mountable",
+                    op, path
                 )));
             }
             Ok(fs)
         }
     }
 
     fn get_rotational_info(fsname: &str) -> Result<String, ConfigError> {
+        let op = "data-dir.rotation.get";
```

Comment:
seem rotation is not related to data-dir?

Reply:
It is only called by check_data_dir right now, so this matches the code. I agree that this relationship is not guaranteed in code and it is possible for this to get out of sync.